Model/ResourceModificationTimeRepository.php

```diff
@@ -13,20 +13,21 @@ public function __construct(Context $dbContext, $connectionName = null)
         $this->dbConnection = $dbContext->getResources()->getConnection($connectionName);
     }
 
-    public function getLastModificationTime(string $identifier)
+    public function getLastModification(string $identifier)
```

Comment:
Given that we have `X-Message-Group-Id`, shouldn't this method be called `getMessageGroup`? And shouldn't the same noun be used elsewhere (e.g. in the table name)?